docs/content/preview/deploy/multi-dc/async-replication/async-transactional-setup-dblevel.md

```diff
@@ -0,0 +1,232 @@
+---
+title: Set up transactional xCluster replication
+headerTitle: Set up transactional xCluster replication
+linkTitle: Set up replication
+description: Setting up transactional (active-standby) replication between universes
+headContent: Set up unidirectional transactional replication
+menu:
+  preview:
+    parent: async-replication-transactional
+    identifier: async-transactional-setup-1-db
+    weight: 10
+badges: ysql
+type: docs
+---
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+  <li >
+    <a href="../async-transactional-setup-dblevel/" class="nav-link active">
+      <i class="icon-shell"></i>
+      Semi-Automatic Setup
+    </a>
+  </li>
+  <li >
+    <a href="../async-transactional-setup/" class="nav-link">
+      <i class="icon-shell"></i>
+      Manual Setup
+    </a>
+  </li>
+</ul>
+
+Semi-automatic setup of transactional xCluster replication simplifies the operational complexity of managing replication and making DDL changes.
```

Comment:
I see 6 more references to `Semi-automatic setup`. Can we rename them all to `Semi-automatic mode`?